gallery/src/pages/StylingComponents.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {GalleryTile} from '../gallery-components/GalleryTiles';
+
+const StylingRSCPage = () =>
+  <GalleryTile title="How to style your app when using RSC">
+    <p className="nx-p">
+      If you have styling needs that aren't covered by the styles included in React Shared Components (RSC),
+      then you're going to have to create your own styles.
+    </p>
+    <h2 className="h2">Page specific stylesheets</h2>
```

Comment:
Should be `<h3>`s and use tile subsection styles

Reply:
> Should be `<h3>`s and use tile subsection styles

I went so far as to use the subsection HTML from RSC-155: https://github.com/sonatype/sonatype-react-shared-components/pull/72/commits/b3c842e717876bc3dfea76fdddb05203eadf2e55

Reply:
> I went so far as to use the subsection HTML from RSC-155: b3c842e

:+1: I will however want RSC-155 to be merged first then so we can verify this page looks ok with those styles, especially in relation to that `<h4>` which isn't something we have demonstrated on the nx-tile page.